src/libbson/src/bson/bson.c

```diff
@@ -127,7 +127,7 @@ _bson_impl_inline_grow (bson_impl_inline_t *impl, /* IN */
 
    req = bson_next_power_of_two (impl->len + size);
 
-   if (req <= INT32_MAX) {
```

Comment:
I think this is a reasonable change, since BSON_MAX_SIZE better conveys what this should be checking. As a sanity check, `BSON_MAX_SIZE` is equal to `INT32_MAX` right? There should be no observable change here.

Reply:
Yes, BSON_MAX_SIZE is equal to INT32_MAX. This changed line will not change the behaviour of the code.